core/src/main/java/org/apache/carbondata/core/scan/filter/executer/RowLevelFilterExecuterImpl.java

```diff
@@ -190,6 +191,9 @@ private void initMeasureChunkIndexes() {
   @Override
   public BitSetGroup applyFilter(RawBlockletColumnChunks rawBlockletColumnChunks,
       boolean useBitsetPipeLine) throws FilterUnsupportedException, IOException {
+    if (exp instanceof MatchExpression) {
```

Comment:
after this change, please modify the testcase LuceneFineGrainDataMapSuite to assert the result